src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/workflow/Workflow.kt

```diff
@@ -15,6 +22,16 @@ class Workflow(val manager: WorkflowManager, val id: String) {
         }
     )
 
+    val model by lazy {
+        repository.node(WORKFLOW_MODEL_ROOT) { query { name(id) } }
+            .firstOrNull() ?: throw WorkflowException("No workflow model found at '$WORKFLOW_MODEL_ROOT!'")
+    }
+
+    val resourceType = common.obj.string {
+        convention(ResourceType.ASSET.value)
```

Comment:
when page need to be reviewed then the workflow resource type is page not asset.
this should not be here as default.

Workflow task could have such default but this one here is rather wrong.

remove that property please